maxim/maxim.py

```
@@ -945,15 +948,19 @@ def check():
-                    scribe().warning(f"[MaximSDK] Log repository not found: {logger.id}. We will be dropping all logs.")
+                    scribe().warning(
+                        f"[MaximSDK] Log repository not found: {logger.id}. We will be dropping all logs."
+                    )
                     if self.raise_exceptions:
                         raise ValueError(f"Log repository not found: {logger.id}")
                     return
                 scribe().debug(f"[MaximSDK] Log repository found: {logger.id}")
             except Exception as e:
-                scribe().error(f"[MaximSDK] Failed to check repository existence: {str(e)}")
+                scribe().error(
+                    f"[MaximSDK] Failed to check repository existence: {str(e)}"
```

Ruff (0.12.2)

Use explicit conversion flag

Replace with conversion flag

(RUF010)